types/big.js/big.mjs.d.ts

```diff
@@ -0,0 +1,3 @@
+export * from './';
+import Big from '/';
```

Comment:
Typo?  It's unusual to import from an absolute path.

Reply:
You're absolutely right, fixed